style.css

```diff
@@ -11,8 +11,15 @@ body {
 }
 
 /* TOP NAV  */
-
 header {
+  position: fixed;
```

Comment:
this is the substantive change, everything else is just renaming things and moving around code